tasks/test/mocha-chrome.js

```diff
@@ -55,17 +57,26 @@ module.exports = function(grunt) {
           );
 
           await new Promise(async (resolve, reject) => {
-            eventHandler1.on('mocha:end', stats => {
-              if (stats.failures) reject(stats);
-              else resolve(stats);
+            eventHandler1.on('mocha:end', async results => {
+              const { stats, coverage } = results;
+              if (stats.failures) {
+                reject(stats);
+              } else {
+                if (coverage) {
+                  await fs.mkdir('./.nyc_output/', { recursive: true });
+                  await fs.writeFile(
```

Comment:
Any errors in these will not be caught properly and might be problematic - I would probably consider doing something like
```js
          const { stats, coverage }  = await new Promise(resolve => eventHandler1.on('mocha:end', resolve));
          if (stats.failures) {
            throw new Error(stats);
          }
          if (coverage) {
            await fs.mkdir('./.nyc_output/', { recursive: true });
            await fs.writeFile(
              './.nyc_output/out.json',
              JSON.stringify(coverage)
            );
          }
```

Reply:
how does this look now? i tried the above but had some issues so i tried to find something between that and the current structure.